SCons/Taskmaster.py

```diff
@@ -63,6 +63,8 @@
 NODE_EXECUTED = SCons.Node.executed
 NODE_FAILED = SCons.Node.failed
 
+display = SCons.Util.display
```

Comment:
There's no `def display()` in SCons/Util.py...

Reply:
I copied this from Main.py, display is a global scope var there

Reply:
gotcha. I guess I missed it.